ui/tests/unit/routes/vault/cluster/oidc-callback-test.js

```diff
@@ -0,0 +1,111 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import sinon from 'sinon';
+
+module('Unit | Route | vault/cluster/oidc-callback', function (hooks) {
+  setupTest(hooks);
+  const parentNs = 'admin';
+  const childNs = 'admin/child-ns';
+  const path = 'oidc';
+  const customPath = 'oidc-dev';
+  const code = 'lTazRXEwKfyGKBUCo5TyLJzdIt39YniBJOXPABiRMkL0T';
+  const state = (ns) => {
+    ns ? 'st_91ji6vR2sQ2zBiZSQkqJ' + `,ns=${ns}` : 'st_91ji6vR2sQ2zBiZSQkqJ';
+  };
+
+  hooks.beforeEach(function () {
```

Comment:
We should probably clean up the stub in a afterEach:
`hooks.afterEach(function() {
this.windowStub.restore();
}`